### ISSUE TITLE
Can't install Minecraft 1.8

### ISSUE THREAD

**Issue by JakobDev** [tomsik68/mclauncher-api#55]
If I run minecraftLauncherBackend.updateMinecraft("1.8", null) I get this error:
```
/usr/lib/jvm/java-8-openjdk-amd64/bin/java -javaagent:/snap/intellij-idea-community/152/lib/idea_rt.jar=33832:/snap/intellij-idea-community/152/bin -Dfile.encoding=UTF-8 -classpath /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/charsets.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/cldrdata.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/dnsns.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/icedtea-sound.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/jaccess.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/localedata.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/nashorn.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/sunec.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/sunjce_provider.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/sunpkcs11.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/zipfs.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/jce.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/jsse.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/management-agent.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/resources.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/rt.jar:/home/jakob/zw/mclauncher-cmd/out/production/mclauncher-cmd:/home/jakob/Downloads/mclauncher-api-0.3.4.jar:/home/jakob/Downloads/json-smart-1.1.1.jar jakobdev.mclauncher_cmd.mclauncher_cmd install 1.8
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking com.mojang:netty:1.6
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking java3d:vecmath:1.5.2
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking net.sf.trove4j:trove4j:3.0.3
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking com.ibm.icu:icu4j-core-mojang:51.2
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking net.sf.jopt-simple:jopt-simple:4.6
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking com.paulscode:codecjorbis:20101023
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking com.paulscode:codecwav:20101023
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking com.paulscode:libraryjavasound:20101123
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking com.paulscode:librarylwjglopenal:20100824
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking com.paulscode:soundsystem:20120107
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking io.netty:netty-all:4.0.15.Final
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking com.google.guava:guava:17.0
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking org.apache.commons:commons-lang3:3.3.2
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking commons-io:commons-io:2.4
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking commons-codec:commons-codec:1.9
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking net.java.jinput:jinput:2.0.5
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking net.java.jutils:jutils:1.0.0
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking com.google.code.gson:gson:2.2.4
Jul 16, 2019 9:09:59 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking com.mojang:authlib:1.5.21
Jul 16, 2019 9:10:00 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking com.mojang:realms:1.6.1
Jul 16, 2019 9:10:00 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking org.apache.commons:commons-compress:1.8.1
Jul 16, 2019 9:10:00 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking org.apache.httpcomponents:httpclient:4.3.3
Jul 16, 2019 9:10:00 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking commons-logging:commons-logging:1.1.3
Jul 16, 2019 9:10:00 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking org.apache.httpcomponents:httpcore:4.3.2
Jul 16, 2019 9:10:00 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking org.apache.logging.log4j:log4j-api:2.0-beta9
Jul 16, 2019 9:10:00 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking org.apache.logging.log4j:log4j-core:2.0-beta9
Jul 16, 2019 9:10:00 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking org.lwjgl.lwjgl:lwjgl:2.9.1
Jul 16, 2019 9:10:00 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking org.lwjgl.lwjgl:lwjgl_util:2.9.1
Jul 16, 2019 9:10:00 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking org.lwjgl.lwjgl:lwjgl-platform:2.9.1
Jul 16, 2019 9:10:00 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller downloadLibrary
INFORMATION: Downloading library /home/jakob/.minecraft/libraries/org/lwjgl/lwjgl/lwjgl-platform/2.9.1/natives.jar
Jul 16, 2019 9:10:01 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking net.java.jinput:jinput-platform:2.0.5
Jul 16, 2019 9:10:01 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller downloadLibrary
INFORMATION: Downloading library /home/jakob/.minecraft/libraries/net/java/jinput/jinput-platform/2.0.5/natives.jar
Jul 16, 2019 9:10:01 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking tv.twitch:twitch:6.5
Jul 16, 2019 9:10:01 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller install
INFORMATION: Checking tv.twitch:twitch-platform:6.5
Jul 16, 2019 9:10:01 PM sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller downloadLibrary
INFORMATION: Downloading library /home/jakob/.minecraft/libraries/tv/twitch/twitch-platform/6.5/natives.jar
Exception in thread "main" java.lang.NullPointerException
	at sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller.downloadLibrary(MCDownloadVersionInstaller.java:155)
	at sk.tomsik68.mclauncher.impl.versions.mcdownload.MCDownloadVersionInstaller.install(MCDownloadVersionInstaller.java:98)
	at sk.tomsik68.mclauncher.backend.MinecraftLauncherBackend.updateMinecraft(MinecraftLauncherBackend.java:62)
	at jakobdev.mclauncher_cmd.mclauncher_cmd.main(mclauncher_cmd.java:27)

Process finished with exit code 1
```

**Comment by tomsik68** [comment]
Fixed by 48d081b943b4895763032819d28588ae1816a4d9. Please re-open if the issue persists with mclauncher-api v0.3.5. Before testing, please delete your local minecraft instance folder. In particular, the local version JSON could've been incorrectly parsed by the previous versions causing newer versions to re-use an incorrect file.

**Comment by JakobDev** [comment]
Have you a download link for the 0.3.5.jar?

**Comment by tomsik68** [comment]
I've just uploaded it: https://raw.githubusercontent.com/tomsik68/maven-repo/master/sk/tomsik68/mclauncher-api/0.3.5/mclauncher-api-0.3.5.jar

**Comment by JakobDev** [comment]
[It still does not work](https://pastebin.com/SN3afXrB)

**Comment by tomsik68** [comment]
It's weird because the twitch-platform library shouldn't be downloaded on Linux according to rules. Did you delete the `/tmp/mcins` folder before trying it?

**Comment by JakobDev** [comment]
This folder does not exist before.

**Comment by tomsik68** [comment]
Should be fixed since 86fd9aa49ad8b6dc84468fc3b473803670f28c0c . Please re-open if the issue persists.
